rules/CodeQuality/Naming/MethodCallToVariableNameResolver.php

```diff
@@ -98,8 +98,17 @@ private function getVariableName(MethodCall $methodCall, string $methodCallVarNa
         return $argumentName . ucfirst($variableName);
     }
 
+    private function isNamespacedFunctionName(string $functionName): bool
+    {
+        return str_contains($functionName, '\\');
+    }
+
     private function getFallbackVarName(string $methodCallVarName, string $methodCallName): string
```

Comment:
This should be renamed to avoid confussion:

```suggestion
    private function getFallbackVarName(string $callerName, string $methodCallName): string
```

Also in the all methods above. There can be a variable, function call, property...

Reply:
ok, implemented https://github.com/rectorphp/rector-src/pull/251/commits/5dcfddd40a4858dc3353bdcd89ef86b37e78c452

Reply:
Should be exactly `callerName`, as caller could mean node as well

Reply:
Oops, should be fixed now https://github.com/rectorphp/rector-src/pull/253